### DISCUSSION TITLE
How to change Google account

### DISCUSSION THREAD

**Discussion by Rinbad** [phw198/OutlookGoogleCalendarSync#1944] [Q&A]
I want to use OGCS with a different Google account from the one I set it up with, but there seems to be no way of making a change (and I neither need nor want to go down the multiple instances road). The first-chosen account simply will not delete to allow another to be substituted. I've tried uninstall (then clean registry, then re-boot) and re-install, but it seems impossible to get a proper clean uninstall and the memory remains in the new install of the former settings! Is there any way of wiping the slate clean and starting afresh, or is this program unchangeable for ever once it has been used with one particular Google account? Any help would be very much appreciated — I wasted a couple of hours trying to work out a way myself, but with total lack of success. What am I missing?

**Comment by phw198** [comment]
It should just be a case of clicking Disconnect Account on the Google settings tab, but I think there is a bug and this is not happening.

Until I get a fix out, after clicking that button, you just need to manually delete the file (in the same folder as your logs and settings) that starts with Google and ends with "token" (I think - I'm not at my computer at the moment).

**Reply by Rinbad** [reply]
Thanks for such a quick reply. Manual deletion worked and I got a "blank sheet" to start on. "Retrieve calendars" duly brought up the Google accounts, from which I was able to choose the one I wanted, and this now figures in "Connected Account". However, the "Select Calendar" field does not populate and there's an error-message "Failed to retrieve" referring me to "Sync" for more information. There I find the following: "Unable to get the list of Google calendars. The following error occurred. / Request had insufficient authorisation. [403=Forbidden]" This certainly didn't happen a few years back when I set OGCS up for the first time. Again, any help would be much appreciated.

**Reply by phw198** [reply]
That's the same issue as #1937
